packages/guides/src/NodeRenderers/Html/SpanNodeRenderer.php

```diff
@@ -14,11 +14,15 @@
 namespace phpDocumentor\Guides\NodeRenderers\Html;
```

Comment:
This class needs our attention :-) Somehow I think we should transformate the spannode into a proper node structure like all other nodes. That would make it possible to use the logic of renderers and templates for spans. It would remove a lot of weird code from the project. 

Let's discuss this after we merged this PR. Introducing all the templates is already a really great move!

Reply:
It certainly does